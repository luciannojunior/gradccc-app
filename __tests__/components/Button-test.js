import 'react-native';
import React from 'react';
import Button from '../../src/components/Button';

import renderer from 'react-test-renderer';

it('Button renders correctly', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});