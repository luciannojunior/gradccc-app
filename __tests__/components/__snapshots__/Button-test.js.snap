// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#069",
      "borderColor": "#d6d7da",
      "borderRadius": 4,
      "borderWidth": 2,
      "height": 64,
      "margin": 10,
      "opacity": 1,
      "padding": 20,
      "width": 140,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "alignSelf": "center",
        "color": "#fff",
        "fontSize": 18,
      }
    }
  />
</View>
`;
